<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryForm</class>
 <widget class="QWidget" name="QueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>Very Simple. It only allows one condition. </string>
     </property>
     <property name="title">
      <string>Simple Query Builder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="fieldOnelineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>First Argument</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="operatorComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Pick an operator for the condition</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>&lt;</string>
            </property>
            <item>
             <property name="text">
              <string>&lt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>isNotEqualTo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>isEqualTo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Contains</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BeginsWith</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EndsWith</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DoesNotContain</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DoesNotBeginWith</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DoesNotEndWith</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IsNull</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IsNotNull</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IsEmpty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IsNotEmpty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Between</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NotBetween</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLineEdit" name="fieldTwolineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Second Argument</string>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fieldThreelineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Third Argument</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="whereCheckBox">
          <property name="text">
           <string>WHERE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SELECT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attrLineEdit"/>
        </item>
        <item>
         <widget class="QCheckBox" name="selectIntoCheckBox">
          <property name="toolTip">
           <string>Create a new Table/Relation based on the result of current query</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string>INTO</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="selectIntoLineEdit">
          <property name="toolTip">
           <string>New Table/Relation name</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>FROM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tableLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>Run</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/rec/ico/run.ico</normaloff>:/rec/ico/run.ico</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delTableButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete Table</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="trucTableButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Truncate Table</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
